### PR TITLE
Contributions EUR epic test

### PR DESCRIPTION
**Pull request by svillafe**
# Related PRs:
#19268
#19263

## What does this change?
Add the epic EUR test.

## Does this affect other platforms - Amp, Apps, etc?
No
## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

### Control in EUR country:
picture 581

### Variant in EUR country:
picture 580

## Tested in CODE?
Not yet

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-eur-support.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+} from 'lib/geolocation';
+
+import config from 'lib/config';
+
+import type { CtaUrls } from 'common/modules/commercial/contributions-utilities';
+
+const EURsupportURL = 'https://support.theguardian.com/eu';
+const abTestName = 'AcquisitionsEpicEurSupport';
+
+const oneButtonTemplate = (urls: CtaUrls) => {
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-eur-support.js]: Missing return type 🎉

Comment (svillafe) [reply static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-eur-support.js]: 😅

**Review by jranks123 — APPROVED** [review]
👍

**Comment by svillafe** [comment]
Tested in CODE and all the channels are working

**Review by Mullefa — APPROVED** [review]

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**